pages/guides/integrations/with-formik.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Chakra UI + Formik
+description: Integrating Chakra UI with Formik
+tags: ['form', 'formik']
+author: nikolovlazar
+---
+
+The way we start building forms with [Formik](https://formik.org/) is by using
+its `useFormik` hook that returns us the formik instance. That Formik instance
+contains pretty much everything we need to connect our form's UI elements and
+submit handler. Let's see a pure React + Formik example:
+
+```typescript
+import React from 'react'
+import { useFormik } from 'formik'
+
+const SignupForm = () => {
+  // Pass the useFormik() hook initial form values and a submit function that will
+  // be called when the form is submitted
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+    },
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2))
+    },
+  })
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <label htmlFor='email'>Email Address</label>
+      <input
+        id='email'
+        name='email'
+        type='email'
+        onChange={formik.handleChange}
+        value={formik.values.email}
+      />
+
+      <button type='submit'>Submit</button>
+    </form>
+  )
+}
+```
```

Comment:
I'd move this iinto a Sandpack aswell just to make it more consistent over the page. This would also show the visual difference between plain react and chakra ui.

Reply:
Done!